netlify.toml

```
@@ -2,7 +2,7 @@
     publish = "public"
 
 [build.environment]
-    HUGO_VERSION = "0.124.1"
+    HUGO_VERSION = "0.125.0"
```

Newer release. Not sure how quickly they will continue.
```suggestion
    HUGO_VERSION = "0.125.1"
```

---

Done